packages/react-component-library/src/components/Timeline/Timeline.stories.tsx

```diff
@@ -414,6 +419,55 @@ export const WithCustomColumns = () => {
 WithCustomColumns.parameters = disableScrollableRegionFocusableRule
 WithCustomColumns.storyName = 'With custom columns'
 
+export const WithCustomRowCss = () => {
+  const rowCss = css`
+    height: 40px;
+  `
+  const rowContentProps = {
+    css: css`
+      background-color: ${ColorNeutral100};
+    `,
+  }
+  const rowHeaderProps = {
+    css: css`
+      background-color: ${ColorNeutral200};
+    `,
+  }
+
+  return (
+    <Timeline
+      hasSide
+      startDate={new Date(2020, 3, 1)}
+      today={new Date(2020, 3, 15)}
+    >
+      <TimelineTodayMarker />
+      <TimelineMonths />
+      <TimelineWeeks />
+      <TimelineDays />
+      <TimelineRows>
+        <TimelineRow
+          css={rowCss}
```

Comment:
The general concept looks good to me :+1: 

Could you check if the root `css` prop on TimelineRow is working when the component library is used in another project? 

I gave it a go and the content and header ones were fine, but the row height was unchanged (using this same example).

Reply:
Just retested – this is fixed now :slightly_smiling_face: